Makefile

```diff
@@ -0,0 +1,49 @@
+# Makefile
+
+.PHONY: pytest pytest-docker-build pytest-docker-run dvc-local-remote repo-docker-build
+
+# Build Docker image from Dockerfile to run the repository
+repo-docker-build:
+	docker image rm -f $(DMD_ERA5_IMAGE_NAME_REPO)
+	docker build -t $(DMD_ERA5_IMAGE_NAME_REPO) -f Dockerfile.repo .
+
+# Run tests except those marked as "docker"
+pytest:
+	pytest -v
+
+# Build Docker image from Dockerfile to run tests marked as "docker"
+pytest-docker-build:
+	docker image rm -f $(DMD_ERA5_IMAGE_NAME_PYTEST)
+	docker build -t $(DMD_ERA5_IMAGE_NAME_PYTEST) -f Dockerfile.pytest .
+
+# Run tests marked as "docker" in Docker container
+pytest-docker-run:
+	docker run --rm $(DMD_ERA5_IMAGE_NAME_PYTEST)
```

Comment:
yes, could be done, and it would definitely be a cheaper setup for testing. However, given how the package works currently, I see one main disadvantages to doing that: you currently need to run the tool from the root of the repo, and cannot choose where the downloaded data will be saved. Additionally, I'm personally not a big fan of having tests creating/deleting files on my local file system. Will open an issue about this in any case, and can be looked at in the future.

Reply:
No need to block the merge given that this way works, we can discuss on the issue.